src/app/features/Chanels/redux/actions.js

```diff
@@ -0,0 +1,32 @@
+import { GET_CHANNELS, GET_POSTS } from './actionsTypes';
+import { channelsMockData, postsMockData } from '../../../utils/mockData';
+
+export const getChannels = (channels) => ({
+  type: GET_CHANNELS,
+  payload: channels,
+});
+
+export const getPosts = (posts) => ({
+  type: GET_POSTS,
+  payload: posts,
+});
+
+export function fetchChannels() {
+  return async (dispatch) => {
+    try {
+      dispatch(getChannels(channelsMockData));
```

Comment:
try to make asyn channels mock service to emulate async call. Currently, this code is synchronous and redux-thunk becomes obsolete